src/components/molecules/TableComponent/TableComponent.tsx

```diff
@@ -15,55 +33,149 @@ const TableComponent: React.FunctionComponent<TableComponentPropsType> = ({
+    "TableComponent__item-wrapper",
+    itemWrapperStyles
+  );
+
+  return (
+    <div className={`TableComponent ${table.reference}`}>
+      <div className={itemClasses}>
+        <div className="TableComponent__occupancy-warning">
           {locked ? "locked" : full ? "full" : ""}
         </div>
-        <div className="table-number">{table.title}</div>
+        <div className={itemWrapperClasses}>
+          {table.title}
 
-        {users &&
-          users.length >= 0 &&
-          users.map((user) => (
+          {isRemoveButtonShown && (
             <img
-              onClick={() => openUserProfileModal(user.id)}
-              key={user.id}
-              className="profile-icon table-participant-picture"
-              src={(!user.anonMode && user.pictureUrl) || DEFAULT_PROFILE_IMAGE}
-              title={(!user.anonMode && user.partyName) || DEFAULT_PARTY_NAME}
-              alt={`${
-                (!user.anonMode && user.partyName) || DEFAULT_PARTY_NAME
-              } profile`}
-              width={imageSize}
-              height={imageSize}
+              className="TableComponent__delete-icon"
+              src={PortalCloseIcon}
+              alt="remove table"
+              onClick={toggleModal}
             />
-          ))}
-        {users &&
-          table.capacity &&
-          table.capacity - users.length >= 0 &&
-          [...Array(table.capacity - users.length)].map((e, i) => (
-            <span
-              key={i}
-              onClick={() => onJoinClicked(table.reference, locked)}
-              id={`join-table-${venue?.name}-${table.reference}`}
-              className="add-participant-button"
-            >
-              +
-            </span>
-          ))}
+          )}
+        </div>
+        <div className="TableComponent__users">
+          {users &&
+            users.length >= 0 &&
+            users.map((user) => (
+              <img
+                onClick={() => openUserProfileModal(user.id)}
+                key={user.id}
+                className="TableComponent__profile-icon"
+                src={
+                  (!user.anonMode && user.pictureUrl) || DEFAULT_PROFILE_IMAGE
+                }
+                title={(!user.anonMode && user.partyName) || DEFAULT_PARTY_NAME}
+                alt={`${
+                  (!user.anonMode && user.partyName) || DEFAULT_PARTY_NAME
+                } profile`}
+                width={imageSize}
+                height={imageSize}
+              />
+            ))}
+          {users &&
+            table.capacity &&
+            table.capacity - users.length >= 0 &&
+            [...Array(table.capacity - users.length)].map((e, i) => (
+              <span
+                key={i}
+                onClick={() => onJoinClicked(table.reference, locked)}
+                id={`join-table-${venue?.name}-${table.reference}`}
+                className="TableComponent__add-user"
+              >
+                +
```

Comment:
Looks like out of place. Something like `{STRING_PLUS}` (can check `interpolationSettings.ts`) might make for clearer intent (if FA icon isn't that necessary)

Reply:
replaced with `string_plus` constant 2ba4481